README.md

```diff
@@ -93,22 +93,21 @@ As a first step you'll need to:
    - In the manifest editor, set the ``allowPublicClient`` property to **true**
    - Click on **Save** in the bar above the manifest editor.
 1. In the app's registration screen, click on the **Authentication** blade in the left and:
-   - In the **Suggested Redirect URIs for Public Clients(mobile, desktop)** select the check box of **https://login.microsoftonline.com/common/oauth2/nativeclient**.
-   - Click on **Save** in the bar above.
-1. In the app's registration screen, click on the **API permissions** blade in the left to open the page where we add access to the Apis that your application needs.
-   - Click the **Add a permission** button and then,
-   - Ensure that the **Microsoft APIs** tab is selected.
-   - In the *Commonly used Microsoft APIs* section, click on **Microsoft Graph**
-   - In the **Delegated permissions** section, select the **User.Read**, **User.ReadBasic.All** in the list. Use the search box if necessary.
-   - Click on the **Add permissions** button at the bottom.
+   - In the **Platform configurations** section select **Add a platform** and create a new **Web** application
```

Comment:
We should change this to create a "Mobile and desktop applications", and it's a command line app.

Reply:
Just changed that to say "mobile and desktop", and adjusted where sample_cache.json is located so Maven pulls it into the .jar automatically.